article/models/article.py

```diff
@@ -58,6 +58,9 @@ def validate_challenge(self):
         if self.challenge.is_ended:
             raise ValidationError({"challenge": "종료한 챌린지입니다."})
 
+        if self.challenge.is_start:
```

Comment:
조건이 반대가 되어야 하는거아니에요?
```suggestion
        if not self.challenge.is_start:
```

Reply:
음 정상작동해서 이상한 점을 몰랐네요
함수명을 is_not_start로 바꾸던지, is_start 함수의 로직을 바꿀게욤...